Modules/add&Remove.js

```diff
@@ -0,0 +1,58 @@
+import { bookContainer, bookTitle, bookAuthor } from './variables.js';
+
+let bookList = [];
+
+export const bookUpload = () => {
+  bookContainer.innerHTML = '';
+  for (let i = 0; i < bookList.length; i += 1) {
+    bookContainer.innerHTML += `
+      <div class="books">
+        <div class ="book-wrapper"> 
+          <div class="book-holder">
+              <h3>"${bookList[i].title}"</h3>
+              <h3>by</h3>
+              <h3>${bookList[i].author}</h3>
+          </div>   
+          <button onclick="DisplayBook.deleted(${i})" id="removeBtn" >Remove</button>
+        </div>   
+          <hr />
+      </div>
+     `;
+    bookTitle.value = '';
+    bookAuthor.value = '';
+  }
+};
+
+const bookRemove = document.querySelectorAll('#removeBtn');
+
+bookRemove.forEach((button) => {
+  button.addEventListener('click', () => {
+    const dataSet = parseInt(button.dataset.id, 8);
+    const buttonId = bookList.findIndex((object) => object.id === dataSet);
+    const deleted = (index) => {
+      bookList.splice(index, 1);
+      bookUpload();
+      localStorage.setItem('bookList', JSON.stringify(bookList));
+    };
+    deleted(buttonId);
+  });
```

Comment:
You did a great job by adding the function to delete the book but unfortunately, the user interface is not working as expected I'm not able to add the book. please fix that it is also part of the requirement : +1: 

I suggest you add the button which allow you to delete books check the example in the design below 

![image](https://user-images.githubusercontent.com/95297251/206204288-e95296c1-28a6-4c27-ac4b-75708d3b1214.png)